packages/mobile/src/recipients/RecipientItem.tsx

```diff
@@ -14,7 +14,9 @@ interface Props {
 
 class RecipientItem extends React.PureComponent<Props> {
   onPress = () => {
-    this.props.onSelectRecipient(this.props.recipient)
+    const recipient = this.props.recipient
+    recipient.address = recipient.address?.toLowerCase()
+    this.props.onSelectRecipient(recipient)
```

Comment:
This is dangerous! We're mutating an existing object. It's very likely this comes from the redux state and we're directly mutating it, bypassing the immutability assumptions of the redux state.

Reply:
Addressed on https://github.com/celo-org/celo-monorepo/pull/4979